### ISSUE TITLE
SmartSelection only works on first file diff

### ISSUE THREAD

**Issue by campersau** [rtfpessoa/diff2html#303]
### Step 0: Describe your environment

- OS: Windows 10
- diff2html version: 3.0.2
- Using diff2html directly or using diff2html-ui helper: diff2html-ui helper
- Extra flags: `smartSelection: true`

### Step 1: Describe the problem:

Copy & paste copies line numbers from diffs but not from the first one.

#### Steps to reproduce:

1. Go to the demo page (link below)
2. Select and copy text from the first diff -> no line numbers are selected and copied.
3. Select and copy text from the diffs after the first one -> line numbers are selected and copied.

#### diff example:

https://diff2html.xyz/demo.html?matching=none&matchWordsThreshold=0.25&maxLineLengthHighlight=10000&diffStyle=word&renderNothingWhenEmpty=0&matchingMaxComparisons=2500&maxLineSizeInBlockForComparison=200&outputFormat=line-by-line&drawFileList=1&synchronisedScroll=1&highlight=1&fileListToggle=1&fileListStartVisible=0&smartSelection=1&diff=https://github.com/rtfpessoa/diff2html/pull/106

#### Observed Results:

- `smartSelection` is only working on the first diff.

#### Expected Results:

- `smartSelection` should work for all diffs.

#### Relevant Code:

Looks like `smartSelection` only gets the first `d2h-diff-table`
https://github.com/rtfpessoa/diff2html/blob/4e28c5c3275f6f703a6923d23d001bfb676331a6/src/ui/js/diff2html-ui-base.ts#L161

Smart selection could even be done with pure CSS, in ungit we are using the following CSS:
```css
.d2h-code-linenumber, .d2h-code-side-linenumber, .d2h-code-line-prefix {
  user-select: none;
}
```
Which also disables selection for the line prefix (`+`, `-`).

---

It also only seems to work for the left side in a `side by side` diff.

**Comment by rtfpessoa** [comment]
@campersau thanks for the suggestion.
I had this https://flaviocopes.com/css-disable-text-selection/ in my todo list but somehow it got lost.
I will work on it.

**Comment by rtfpessoa** [comment]
Released as version 3.1.0
